src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -291,5 +306,38 @@ private void ApplyConfiguration(
         {
             throw new NotSupportedException($"OtlpExporterOptionsConfigurationType '{configurationType}' is not supported.");
         }
+
+#if NET
+        // Apply mTLS configuration from environment variables
+        this.ApplyMtlsConfiguration(configuration);
```

Comment:
I do not want to block this PR, consider adding also support for OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE and other signal specific vars.

There is a chance that cover all cases will required breaking changes (signal specific OTLP options), but the _best effort_ should be also possible.

I have seen at least one method guess which env. var. is appropriate.